### DISCUSSION TITLE
Is thread safe?

### DISCUSSION THREAD

**Discussion by FrancescoSaverioZuppichini** [celery/kombu#1557] [Q&A]
Thanks for the amazing library! Is kombu threads safe? Is there are articles/guides about best practices with threads?

Thanks

Francesco

[a comment by an automated account is not shown]

**Comment by auvipy** [comment]
please open a discussion, instead of creating an issue for this type of questions.

**Reply by FrancescoSaverioZuppichini** [reply]
hey @auvipy thanks a lot, apologies. Feel free to close my issue :)

**Reply by auvipy** [reply]
i transferred to conversation so no need to close it.
